### ISSUE TITLE
Running SWORD on gzipped fasta

### ISSUE THREAD

**Issue by alcooj** [rvaser/sword#2]
Good morning, Robert.

I was wondering if there is a way to run SWORD on input fasta database target files that are compressed (*.fa.gz). I've tried process substitution to read the fa.gz database file using zcat, and it returns "No alignments found." Otherwise, running SWORD on the decompressed database file works. Is there a way to avoid the need to decompress *.fa.gz target files?

Thanks.

**Comment by rvaser** [comment]
Hello,
SWORD parses the database several times so it can't read streams. I would have to refactor the code to enable compressed files but haven't got the time at the moment. Is it crucial for you to use compressed files?

Best regards,
Robert

**Comment by alcooj** [comment]
Not really. It's more of a space-saving issue as some of the reference database files I work with are large.  For now, I'll then just have to work with the decompressed databases.

I appreciate the quick reply and thanks again!
